src/server/pps/server/api_server.go

```diff
@@ -1623,17 +1622,6 @@ func (a *apiServer) sudo(ctx context.Context, f func(*client.APIClient) error) e
 	return f(superUserClient)
 }
 
-// sudoTransaction is a convenience wrapper around sudo for api calls in a transaction
-func (a *apiServer) sudoTransaction(txnCtx *txncontext.TransactionContext, f func(*txncontext.TransactionContext) error) error {
```

Comment:
there's still a use of `sudo` in the pipeline controller for `getPipelineInfo`, I haven't actually checked if that can be removed, too

Reply:
nevermind